docs/manifest.json

```diff
@@ -173,6 +173,24 @@
 		"markdown_source": "../docs/how-to-guides/platform/custom-block-editor/tutorial.md",
 		"parent": "custom-block-editor"
 	},
+	{
+		"title": "Create your First App with Gutenberg Data",
+		"slug": "data-basics",
+		"markdown_source": "../docs/how-to-guides/data-basics/README.md",
+		"parent": "how-to-guides"
+	},
+	{
+		"title": "Setup",
+		"slug": "1-setup",
```

Comment:
I don't know all the rules about how these slugs work, but I'm sure I remember there have been issues with these slugs in the past not being unique enough (e.g. https://github.com/WordPress/gutenberg/pull/37505).

`1-setup` seems a bit risky as I imagine a lot of tutorials could have a `1-setup`. It might be good to try some sort of namespacing convention.

Reply:
A-ha, TIL. Thank you @talldan! I just updated this PR